services/src/handlers/wfs.rs

```diff
@@ -229,8 +231,8 @@ fn get_capabilities(_request: &GetCapabilities) -> Result<HttpResponse> {
     </ows:OperationsMetadata>
     <FeatureTypeList>
         <FeatureType>
-            <Name>Test</Name>
-            <Title>Test</Title>
+            <Name>{workflow}</Name>
+            <Title>Workflow {workflow}</Title>
             <DefaultCRS>urn:ogc:def:crs:EPSG::4326</DefaultCRS>
```

Comment:
I guess we can just prefix `urn:ogc:def:crs:` for now as it works in all (?) of our use cases. I don't know the example for UTM.

It is described here: https://docs.opengeospatial.org/pol/09-048r5.html#_names_for_epsg_definitions

From my perspective, it would be somewhat great to have finally a fully functional capabilities file.